developer-docs-site/docs/releases/devnet-release.md

```diff
@@ -0,0 +1,29 @@
+---
+title: "Devnet Release"
+slug: "devnet-release"
+---
+
+# Latest Devnet Release
+
+The table below contains latest release information for each Aptos component in the mainnet network. We also suggest you monitor the [#devnet-release](https://discord.com/channels/945856774056083548/956692649430093904) channel of [Aptos Discord](https://discord.gg/aptoslabs) for ongoing updates.
+
+See the [Latest Aptos Releases](./index.md) index for how to subscribe to GitHub notifications and more release-related information.
+
+## Aptos CLI
+
+|Release | Docker Image Tag | Docker SHA-256 | Branch | Commit Hash|
+|---|---|---|---|---|
+|[Aptos CLI Release v1.0.7](https://github.com/aptos-labs/aptos-core/releases/tag/aptos-cli-v1.0.7)| `aptos-cli-v1.0.7` | `?` | ?| `?` |
+
+## Aptos Framework
+
+|Release | Docker Image Tag | Docker SHA-256 | Branch | Commit Hash|
+|---|---|---|---|---|
+|[Aptos Framework Release v1.2.0](https://github.com/aptos-labs/aptos-core/releases/tag/aptos-framework-v1.2.0)| `aptos-framework-v1.2.0` | `?` | ?| `?` |
+
+
+## Aptos Node
+
+|Release | Docker Image Tag | Docker SHA-256 | Branch | Commit Hash|
```

Comment:
Can we keep track here which is devnet testnet and mainnet?

Reply:
> Can we keep track here which is devnet testnet and mainnet?

Thanks! Do the distinct pages for each network accomplish this? Or are you saying we need something about the other networks on each of those pages?
https://deploy-preview-6657--aptos-developer-docs.netlify.app/releases/mainnet-release
https://deploy-preview-6657--aptos-developer-docs.netlify.app/releases/testnet-release
https://deploy-preview-6657--aptos-developer-docs.netlify.app/releases/devnet-release

Thanks again!